vscode-extension/src/extension.ts

```diff
@@ -233,22 +223,64 @@ async function createNewAIConfig(
   const fileContentBuffer = await vscode.workspace.fs.readFile(fileContentPath);
   const initialContent = fileContentBuffer.toString();
 
-  const doc = await vscode.workspace.openTextDocument({
-    content: initialContent,
-    language: mode,
+  const workspaceUri = vscode.workspace.workspaceFolders
+    ? vscode.workspace.workspaceFolders[0].uri
```

Comment:
Since we're just using `workspaceUri.path` later, let's just use `uri.path` here directly. If workspace folder is defined `.uri.fsPath` must exist and can't be undefined: 

<img width="1296" alt="Screenshot 2024-02-24 at 17 26 26" src="https://github.com/lastmile-ai/aiconfig/assets/151060367/0343addc-6cf5-4320-bfb5-bf042bec9995">